nbconvert/exporters/base.py

```diff
@@ -135,6 +136,9 @@ def get_export_names(config=get_config()):
     them as an nbconvert.exporter entrypoint.
     """
     exporters = sorted(entrypoints.get_group_named('nbconvert.exporters'))
+    if os.environ.get("NBCONVERT_DISABLE_CONFIG_EXPORTERS"):
```

Comment:
nit: might be good to have a comment here so people know what this is for, i.e. bypass the (potentially slow) enable/disable check per extension if you know that you're supporting all exporters.

I also wonder if there is somewhere in the docs or config options that we could mention this environment variable otherwise people would only ever know about it if they were familiar with this code. Like maybe somewhere in here:

https://github.com/jupyter/nbconvert/blob/master/docs/source/api/exporters.rst

Though that's just auto-generated module doc.

Looking at https://nbconvert.readthedocs.io/en/latest/config_options.html#exporter-options maybe `Exporter.enabled` should mention this env var? Or https://nbconvert.readthedocs.io/en/latest/external_exporters.html?

A CODEOWNER might have better ideas, but I'd think mentioning it with the `Exporter.enabled` config option would be the most _discoverable_ place for mentioning this variable.

Reply:
To @mriedem 's point, https://nbconvert.readthedocs.io/en/latest/config_options.html#exporter-options would be the best place to document the environment variable action. You can add this note by editing the `CLI Flags and Aliases` section in `autogen_config.py`.

Additionally we should probably log out a quick message when we're disabling config exporters so it's visible that the flag was set when helping future nbconvert issues. Use `from traitlets.log import get_logger` and `get_logger().info("Config exporter loading disabled, no additional exporters will be automatically included.")`